cds_ils/static_pages/search_guide.html

```diff
@@ -0,0 +1,674 @@
+
+<p><strong>Example:</strong> <a href="#"><code>"collision theory"~10</code></a></p>
+
+<h4 id="wildcards">Wildcards</h4>
+
+<p>You can use wildcards in search terms to replace a single character (using <code>?</code> operator) or zero or more characters (using <code>*</code> operator).</p>
+
+<p><strong>Example:</strong> <a href="#"><code>title:(crab cavit*)</code></a></p>
+
+The result will match videos with title referring to cavities also in languages other than english.
+
+<p>Wildcard searches can be slow and should normally be avoided if possible.</p>
+
+<hr />
+
+<h2 id="fieldsreference">Fields reference</h2>
+
+<p>The table below lists the data type of each field. Below is a quick description
+of what each data type means and what is possible.</p>
+
+<ul>
+<li><strong>string</strong>: Field does not require exact match (example field: <code>title</code>).</li>
+
+<li><strong>string (exact)</strong>: Field requires exact match (example field: <code>title</code>).</li>
+
+<li><strong>CV</strong>: Controlled vocabularies but take only a specific list of values.</li>
+
+<li><strong>integer</strong>: A number (range queries possible).</li>
+
+<li><strong>datetime</strong>: An ISO8601 date or datetime, e.g. <code>2020-12-31</code> (range queries possible).</li>
+
+<li><strong>boolean</strong>: <code>true</code> or <code>false</code>.</li>
+</ul>
+
+<p>In case of multiple subfields, (<code>\*</code>) applies to all of them</p>
+
+<h3 id="fieldsreferencecommon">Common fields</h3>
+
+<table><thead><tr>
+<th align="left">Field name</th>
+<th align="left">Type</th>
+<th align="left">Notes</th></tr></thead>
+<tbody>
+
+<tr>
+  <td><code>abstract</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_abstracts</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_identifiers.\*</code></td>
+  <td>string</td>
+  <td>Identifiers: material, scheme, value</td>
+</tr>
+<tr>
+  <td><code>alternative_titles.\*</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_titles.language</code></td>
+  <td>CV</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_titles.source</code></td>
+  <td>string (exact)</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_titles.type</code></td>
+  <td>CV</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>alternative_titles.value</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>created_by.\*</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>created_by.type</code></td>
+  <td>string (exact)</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>created_by.value</code></td>
+  <td>string (exact)</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>edition</code></td>
+  <td>string</td>
+  <td></td>
+</tr>
+<tr>
+  <td><code>extensions</code></td>
```

Comment:
I would remove all extensions for the time being, otherwise they require more explanations